packages/cdktf/lib/terraform-resource.ts

```diff
@@ -13,6 +13,8 @@ export interface ITerraformResource {
   count?: number;
   provider?: TerraformProvider;
   lifecycle?: TerraformResourceLifecycle;
+
+  interpolationForAttribute(terraformAttribute: string): string;
```

Comment:
This line is going to change the visibility of existing `interpolationForAttribute`. I prefer this style to exposing `interpolationPrefix` attribute for example, but let me know if there is any coding policy about this.